plugins/check-pr-tests/commands/check-pr-tests.md

```
@@ -0,0 +1,110 @@
+---
+description: Verify test coverage and CI results for a GitHub PR linked to a Jira issue
+argument-hint: <pr-url> [count] [--execute | --dry-run]
+---
+
+## Name
+
+check-pr-tests:check-pr-tests
+
+## Synopsis
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)